server/models/PlatformSubscription.ts

```diff
@@ -450,6 +454,21 @@ class PlatformSubscription extends Model<
     return PlatformSubscription.createSubscription(collectiveId, newSubscriptionStart.toDate(), plan, opts);
   }
 
+  static getPreferredPlatformPayout = getPreferredPlatformPayout;
+
+  static async chargeExpense(expense: Expense) {
+    const payoutMethod = await expense.getPayoutMethod();
+
+    switch (payoutMethod.type) {
+      case PayoutMethodTypes.STRIPE: {
+        return chargePlatformBillingExpenseWithStripe(expense);
+      }
+      default: {
+        return;
+      }
+    }
+  }
+
```

Comment:
For consistency, I think this function should live in the platform subscription lib.